src/main/java/games/strategy/engine/pbem/TripleAForumPoster.java

```diff
@@ -53,17 +53,15 @@ public boolean postTurnSummary(final String summary, final String title) {
     return false;
   }
 
-  private void post(final CloseableHttpClient client, final String token, String text) throws Exception {
+  private void post(final CloseableHttpClient client, final String token, final String text) throws Exception {
     final HttpPost post = new HttpPost(tripleAForumURL + "/api/v1/topics/" + getTopicId());
     addTokenHeader(post, token);
-    if (m_includeSaveGame && m_saveGameFile != null) {
-      text += uploadSaveGame(client, token);
-    }
     post.setEntity(new UrlEncodedFormEntity(
-        Collections.singletonList(new BasicNameValuePair("content", text)),
+        Collections.singletonList(new BasicNameValuePair("content",
+            text + ((m_includeSaveGame && m_saveGameFile != null) ? uploadSaveGame(client, token) : ""))),
         StandardCharsets.UTF_8));
     HttpProxy.addProxy(post);
-    client.execute(post);
+    client.execute(post).close();
```

Comment:
Just curious if there's a reason we're not checking the response status code here (which is probably why a try-with-resources was never used in the first place)?  Even if we shouldn't fail the TripleA code path if the request didn't succeed, should we at least log it?